db/db_impl.h

```diff
@@ -1678,6 +1678,9 @@ class DBImpl : public DB {
   size_t GetWalPreallocateBlockSize(uint64_t write_buffer_size) const;
   Env::WriteLifeTimeHint CalculateWALWriteHint() { return Env::WLTH_SHORT; }
 
+  Status CreateWAL(uint64_t log_file_num, uint64_t recycle_log_number,
+                   const size_t preallocate_block_size, log::Writer** new_log);
```

Comment:
I don't think this `const` is necessary.

Reply:
Fixed